bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/FindReplaceLogicFindAllStatus.java

```diff
@@ -0,0 +1,35 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Vector Informatik GmbH and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Vector Informatik GmbH - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.ui.internal.findandreplace;
+
+public class FindReplaceLogicFindAllStatus implements IFindReplaceLogicStatus {
```

Comment:
I agree, and have changed the structure of the files.

Reply:
The files are now in `internal.findandreplace.findreplacelogicstatus` - the names are descriptive but rather unwieldy; I do not believe this to become a problem since they are used internally anway.

https://github.com/eclipse-platform/eclipse.platform.ui/commit/b8bf9a7e291031b53cf6666060cfd477933e3126